6.1.1: Add release notes (#2796)
sensu/sensu-docs: content/sensu-go/6.1/release-notes.md

---

## 6.1.1 release notes

**October XX, 2020** &mdash; The latest release of Sensu Go, version 6.1.1, is now available for download.

> This patch release includes a number of bug fixes for `sensuctl prune` and `sensuctl dump`, subscription name validation in the web UI, permissions for PATCH requests, and silence expiration times.

Author: ```suggestion
This patch release includes a number of bug fixes that affect proper hook handling with `sensuctl prune` and `sensuctl dump`, entity creation via `sensuctl create`, form validation for subscription names in the web UI, permissions for PATCH requests, and silence expiration times, among others.
```

See the [upgrade guide][1] to upgrade Sensu to version 6.1.1.

**FIXES:**

- ([Commercial feature][172]) [`sensuctl prune`][192] now properly handles hooks when pruning resources.
- ([Commercial feature][172]) Fixed a bug that returned incorrect `!=` results for label selectors when no labels were defined.
- ([Commercial feature][172]) In the web UI, fixed a bug that could cause a GraphQL `no claims` error when a user's access token was no longer valid instead of displaying the sign-out dialog window.
- ([Commercial feature][172]) In the web UI, form validation for subscription names now matches allowed values.
- Fixed a bug that could cause silences to take too long to expire if the cluster goes through a restart or an etcd election.
- Fixed a bug that prevented sensu-agent from shutting down correctly.

> - Per-entity subscriptions now persist with PATCH requests.

Author: ```suggestion
- Entities are now properly created using `sensuctl create`.
- Per-entity subscriptions now persist with PATCH requests.
```

---

## 6.1.1 release notes

> **October XX, 2020** &mdash; The latest release of Sensu Go, version 6.1.1, is now available for download.

Author: ```suggestion
**October 22, 2020** &mdash; The latest release of Sensu Go, version 6.1.1, is now available for download.
```

> This patch release includes a number of bug fixes that affect proper hook handling with `sensuctl prune` and `sensuctl dump`, entity creation via `sensuctl create`, form validation for subscription names in the web UI, permissions for PATCH requests, and silence expiration times, among others.

Author: ```suggestion
This patch release includes a number of bug fixes that affect proper hook handling with `sensuctl prune` and `sensuctl dump`, entity creation via `sensuctl create`, form validation for subscription names in the web UI, and permissions for PATCH requests, among other fixes.
```

See the [upgrade guide][1] to upgrade Sensu to version 6.1.1.

**FIXES:**

- ([Commercial feature][172]) [`sensuctl prune`][192] now properly handles hooks when pruning resources.
- ([Commercial feature][172]) Fixed a bug that returned incorrect `!=` results for label selectors when no labels were defined.
- ([Commercial feature][172]) In the web UI, fixed a bug that could cause a GraphQL `no claims` error when a user's access token was no longer valid instead of displaying the sign-out dialog window.
- ([Commercial feature][172]) In the web UI, form validation for subscription names now matches allowed values.

> - Fixed a bug that could cause silences to take too long to expire if the cluster goes through a restart or an etcd election.

Reviewer: This bug fix is actually not included in 6.1.1, we will ship that in 6.2.0!

> - Fixed a bug that could cause silences to take too long to expire if the cluster goes through a restart or an etcd election.

Author: ```suggestion
```

- Fixed a bug that prevented sensu-agent from shutting down correctly.
- Entities are now properly created using `sensuctl create`.
- Per-entity subscriptions now persist with PATCH requests.
- Any user with [`update` permissions][190] for a resource can now make PATCH requests for that resource.

> - Sensuctl now resolves symbolic links.

Reviewer: Same as ^, this will only be included in 6.2.0

> - Sensuctl now resolves symbolic links.

Author: ```suggestion
```
